packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -56,7 +58,6 @@ library TaikoData {
     struct BlockMetadataInput {
         bytes32 txListHash;
         address beneficiary;
-        uint32 gasLimit;
```

Comment:
So the consensus is that we will hard-code every L2 block's gasLimit to `blockMaxGasLimit`?

Reply:
I would think `blockMaxGasUsed`?